### PR TITLE
feat(Helm-Chart): Groups same helm chart into single tile

### PR DESCRIPTION
**Pull request by abhinandan13jan**
## Fixes:
https://issues.redhat.com/browse/ODC-4085

## Analysis / Root cause:
Group helm multiple chart versions together in developer catalog

## Solution Description:
Altered normalizeHelm functionality to create single item for all hellm chart versions with same name

## Screenshot

Screenshot from 2020-06-10 18-39-08

After change

Screenshot from 2020-06-10 18-37-52

## Test
Added to catalog test suite

## Browser conformation
Chrome 73

### CODE REVIEW

File: frontend/public/components/catalog/catalog-page.tsx [hunk 1]
```diff
@@ -283,7 +282,15 @@ export const CatalogListPage = withExtensions<CatalogListPageExtensionProps>({
               markdownDescription,
               customProperties,
               href: `/catalog/helm-install?chartName=${chartName}&chartURL=${encodedChartURL}&preselected-ns=${currentNamespace}`,
-            });
+            };
+            const existingChart = normalizedCharts.findIndex((hlc) => hlc.obj?.name === chartName);
+            if (existingChart > -1) {
+              if (normalizedCharts[existingChart].obj?.version < helmChart.obj.version) {
```
[the remark this answers is not in the document]
Comment (abhinandan13jan) [reply frontend/public/components/catalog/catalog-page.tsx]: updated

File: frontend/public/components/catalog/catalog-items.tsx [hunk 2]
```diff
@@ -19,6 +19,8 @@ type Metadata = { uid?: string; name?: string; namespace?: string };
 
 export type Item = {
   obj?: {
+    name?: string;
```
Comment (rohitkrai03) [inline frontend/public/components/catalog/catalog-items.tsx:22]: You don't need this change now.

File: frontend/public/components/catalog/catalog-page.tsx [hunk 3]
```diff
@@ -283,7 +282,25 @@ export const CatalogListPage = withExtensions<CatalogListPageExtensionProps>({
...
               href: `/catalog/helm-install?chartName=${chartName}&chartURL=${encodedChartURL}&preselected-ns=${currentNamespace}`,
-            });
+            };
+
+            // group Helm chart with same name and different version together
+            const existingChartIndex = normalizedCharts.findIndex(
+              (hlc) => hlc.obj?.name === chartName,
+            );
+            if (existingChartIndex > -1) {
+              const versionCompare = helmChart.obj.version.localeCompare(
```
Comment (rohitkrai03) [inline frontend/public/components/catalog/catalog-page.tsx:292]:
```suggestion
              const versionCompare = helmChart.obj?.version?.localeCompare(
```

**Comment by debsmita1** [comment]
verified this locally, works fine
/lgtm

**Pull request merged**